### PR TITLE
Project documentation (kaan.ugur)

### PR DESCRIPTION
**Pull request by kaanmeister**
### Pull Request Overview

This pull request adds/changes/fixes...

### Testing Strategy

This pull request was tested by Kaan Ugur

### TODO or Help Wanted

This pull request still needs...

### Build

- [ ] Ran `npm build`.

### Author

Signed-off by: Kaan UGUR <kaanmeister.ro@gmail.com>

### CODE REVIEW

File: docs/project/2024/kaan.ugur/index.md [hunk 1]
```diff
@@ -0,0 +1,167 @@
+# Project Name
+
+**Rust-powered Portable Smart Summer Hat**
```
Comment (alexandruradovici) [inline docs/project/2024/kaan.ugur/index.md:3]:
```suggestion
# Rust-powered Portable Smart Summer Hat
```

Add a one line description of the project below the title.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

File: docs/project/2024/kaan.ugur/index.md [hunk 2]
```diff
@@ -0,0 +1,166 @@
+# Smart Summer Hat
+
+**Rust-powered Portable Smart Summer Hat**
```
Comment (alexandruradovici) [inline docs/project/2024/kaan.ugur/index.md:3]:
```suggestion
Rust-powered Portable Smart Summer Hat
```

File: docs/project/2024/kaan.ugur/index.md [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+| [Rapspberry Pi Pico H](https://www.raspberrypi.com/documentation/microcontrollers/raspberry-pi-pico.html) | The microcontroller | [42 RON](https://www.optimusdigital.ro/ro/placi-raspberry-pi/12393-raspberry-pi-pico-h.html?search_query=pico+H&results=32) |
+| [BMP 280](https://cdn-shop.adafruit.com/datasheets/BST-BMP280-DS001-11.pdf) | Temperature Sensor | [10 RON](https://www.optimusdigital.ro/ro/senzori-senzori-de-presiune/1777-modul-senzor-de-presiune-barometric-bmp280.html?search_query=bmp280&results=11) |
+| [DC Motor (FAN)](https://datasheetspdf.com/datasheet-pdf/917194/775-5520F-CC.html) | DC Motor for FAN | [10 RON](https://www.bitmi.ro/motor-dc-3v-6v-pentru-proiecte-electronice-10651.html?gad_source=1&gclid=EAIaIQobChMIhsGZpandhQMVEplQBh0msAG5EAQYCCABEgLS5vD_BwE) |
+| [1602 Display](https://html.alldatasheet.com/html-pdf/1574132/CRYSTAIFONTZ/LCD-1602A/1009/1/LCD-1602A.html) | LCD Display | [25 RON](https://www.bitmi.ro/electronica/ecran-lcd1602-cu-modul-i2c-iic-10487.html) |
+| [Jumper Wires (Female - Female)]() | Female - Female Wires for connections | [7 RON](https://www.bitmi.ro/componente-electronice/40-fire-dupont-mama-mama-30cm-10503.html) |
```
Comment (alexandruradovici) [inline docs/project/2024/kaan.ugur/index.md:130]:
```suggestion
| Jumper Wires (Female - Female) | Female - Female Wires for connections | [7 RON](https://www.bitmi.ro/componente-electronice/40-fire-dupont-mama-mama-30cm-10503.html) |
```
 if there is no link

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

File: docs/project/2024/kaan.ugur/index.md (hunk 1 above)
Comment (kaanmeister) [reply docs/project/2024/kaan.ugur/index.md]: I have changed the lines how you suggested and have updated.

File: docs/project/2024/kaan.ugur/index.md (hunk 2 above)
Comment (kaanmeister) [reply docs/project/2024/kaan.ugur/index.md]: I have changed the lines how you suggested and have updated.

File: docs/project/2024/kaan.ugur/index.md (hunk 3 above)
Comment (kaanmeister) [reply docs/project/2024/kaan.ugur/index.md]: I have changed the lines how you suggested and have updated.

**Review by irina-b-dev — APPROVED** [review]

**Pull request merged**